src/scss/3_components/kss-templates/links.twig

```diff
@@ -0,0 +1 @@
+<a class="btn" href="#">This is a link</a>
```

Comment:
probably don't need this twig, it can probably be managed by adding a modifier class to this file `src/scss/1_base/_links.scss` or alternatively keep the commentary in this file, but re-use the `kss-templates/links-a.twig` template referred to `src/scss/1_base/_links.scss`.